commit e85948fd270e24a66335937c572cd1be5a26db2c
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Thu Aug 25 03:27:00 2022 +0200

    disable the ld nonsense that breaks with lld
    
    FAILED: demos/widget-factory/widgetfactory_resources.o
    /usr/bin/ld -z noexecstack -r -b binary demos/widget-factory/widgetfactory.gresource -o demos/widget-factory/widgetfactory_resources.o
    ld: error: target emulation unknown: -m or at least one .o file required

diff --git a/demos/gtk-demo/meson.build b/demos/gtk-demo/meson.build
index 1347baf..0200246 100644
--- a/demos/gtk-demo/meson.build
+++ b/demos/gtk-demo/meson.build
@@ -160,7 +160,7 @@ endif
 
 ld = find_program('ld', required : false)
 
-if not meson.is_cross_build() and build_machine.cpu_family() != 'arm' and build_machine.system() == 'linux' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
+if false
   glib_compile_resources = find_program('glib-compile-resources')
 
   # Create the resource blob
diff --git a/demos/widget-factory/meson.build b/demos/widget-factory/meson.build
index 550eb09..162a66a 100644
--- a/demos/widget-factory/meson.build
+++ b/demos/widget-factory/meson.build
@@ -8,7 +8,7 @@ endif
 
 ld = find_program('ld', required : false)
 
-if not meson.is_cross_build() and build_machine.cpu_family() != 'arm' and build_machine.system() == 'linux' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
+if false
   glib_compile_resources = find_program('glib-compile-resources')
 
   # Create the resource blob
diff --git a/gtk/meson.build b/gtk/meson.build
index 3a8457a..a5aceda 100644
--- a/gtk/meson.build
+++ b/gtk/meson.build
@@ -872,7 +872,7 @@ endif
 
 ld = find_program('ld', required : false)
 
-if not meson.is_cross_build() and build_machine.cpu_family() == 'x86_64' and build_machine.system() == 'linux' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
+if false
   glib_compile_resources = find_program('glib-compile-resources')
 
   # Create the resource blob
